cropper/app/lib/AspectRatio.scala

```diff
@@ -0,0 +1,28 @@
+package lib
+
+import scala.annotation.tailrec
+
+object AspectRatio {
+  case class Ratio(friendly: String, width: Int, height: Int)
+
+  val knownRatios = List(
+    Ratio("5:3", 5, 3),
```

Comment:
You could make friendly automatic, but we might potentially want to add things like "megasuperlandslide".